static/js/src/certified-search-results.js

```diff
@@ -0,0 +1,147 @@
+function clearFilters() {
+  location.assign(`/certified?q=`);
+  return false;
+}
+
+function enableApplyFilters() {
+  const filtersSelected = [];
+  const filters = document.querySelectorAll(".js-enable-apply-filters");
+  filters.forEach((filter) => {
+    filter.addEventListener("change", () => {
+      if (!filtersSelected.includes(filter)) {
+        filtersSelected.push(filter);
+      } else {
+        const index = filtersSelected.indexOf(filter);
+        filtersSelected.splice(index, 1);
+      }
+      const button = document.querySelector(".js-apply-filters");
+      if (filtersSelected.length) {
+        button.removeAttribute("disabled");
+      } else {
+        button.disabled = true;
+      }
+    });
+  });
+
+  function displayFilterCount() {
+    const accordion = document.querySelector(".js-filter-count");
```

Comment:
Sorry, didn't see this before, but indentation seems a bit off here?

Reply:
This function is within the `enableApplyFilters()` function as it needs to access the `filtersSelected` array